packages/ember-glimmer/lib/setup-registry.ts

```diff
@@ -66,6 +68,10 @@ export function setupEngineRegistry(registry: Registry) {
 
   registry.optionsForType('helper', { instantiate: false });
 
+  registry.register('helper:loc', helper(function loc(params) {
```

Comment:
Can you put the implementation in a separate file?

Reply:
Sure! My first instinct is to create a new package called `ember-template` and put it in there, sg?